### ISSUE TITLE
addRequestInterceptor - No direct access to instantiated instance object

### ISSUE THREAD

**Issue by tkambler** [marmelab/restful.js#13]
It doesn't appear that it is currently possible to assign custom methods, etc... to an entity via `addResponseInterceptor`:

```
users.addResponseInterceptor(function(users, headers, method, url) {
	users.forEach(function(user) {
		// user === plain object literal. Shouldn't this be the instantiated entity, as is the case w/ Restangular?
	});
    return users;
});
```

**Comment by tkambler** [comment]
Also needs some form of `resticulateEntity()` (lol) so that you can convert attributes into nested entities / collections in their own right.

**Comment by RobinBressan** [comment]
It is not available right now. Need to think about it.

**Comment by jeromemacias** [comment]
You will be able to do it with this PR: #32, will be available soon.

**Comment by fzaninotto** [comment]
Fixed by #32
